dot/sync/disjoint_block_set_test.go

```diff
@@ -0,0 +1,331 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package sync
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	testBlock1 = &pendingBlock{
+		hash:   common.Hash{1},
+		number: 1,
+	}
+	testBlock10 = &pendingBlock{
+		hash:   common.Hash{10},
+		number: 10,
+	}
+)
```

Comment:
Define these locally in the test where they are needed, not at global scope.

Reply:
Updated.